### DISCUSSION TITLE
【广告】《黑神话：悟空》最终预告 | 8月20日，重走西游

### DISCUSSION THREAD

**Discussion by user1-cloud** [share121/inter-knot#757] [其他]
![93681720c96ce064abda3698769311fc1012b92b](https://github.com/user-attachments/assets/e4b73c44-e399-4593-b24c-14cd79fab061)
### 本片包含首次露出的战斗玩法与剧情动画，亦有新的关卡与角色。
 - 所有演示内容，均为**PC版本实机录制**。

本作将于2024 年 8 月 20 日上午 10 点（UTC+8）在 **PS5 / STEAM / EPIC GAMES STORE / WEGAME** 平台正式解锁游玩。欢迎绳匠们前来体验。

******

 - ### 价格表  
|数字标准版|数字豪华版|实体豪华版|实体收藏版|
|:---:|:---:|:---:|:---:|
|268元|328元|820元|1998元|

******
# [(**现在购买**（￣︶￣）↗　)](https://www.heishenhua.com "直面天命")

**Comment by GanlinSama** [comment]
很很期待住了！！！(☆▽☆)

**Comment by QingMingqwq** [comment]
国产3A的启蒙，很期待这款游戏的表现

**Comment by aaa00073** [comment]
联想到绝区零开服的舆论环境，只能说有点担心黑猴到时候会被骂呀，毕竟黑猴的招式也不算多。

**Reply by c1iied** [reply]
黑猴的背后可是藤子，舆论管控经验很强的，再骂也掀不起什么波澜

**Reply by weixingstar** [reply]
> 黑猴的背后可是藤子，舆论管控经验很强的，再骂也掀不起什么波澜

绝可就惨喽，肯定一堆串子拉踩💩

**Comment by lixianyunyou** [comment]
大的要来了

**Comment by Lynette-CN** [comment]
cy，最喜欢的一集

**Comment by Mayf1ower** [comment]
能在绳网上打广告游科也是好起来了，希望黑猴能狠狠接住这波流量🎉

**Comment by DipleMoss** [comment]
太棒了，这是我最喜欢的一集

**Comment by Lephaethon** [comment]
狠狠期待住了！

**Comment by warning-Kirov-reporting** [comment]
学生党还在观望（悲

**Comment by kaijinti** [comment]
二十号买，但是上班玩不了啊，哭死
